src/main/resources/view/DarkTheme.css

```diff
@@ -1,26 +1,26 @@
 .background {
-    -fx-background-color: derive(#1d1d1d, 20%);
-    background-color: #383838; /* Used in the default.html file */
+    -fx-background-color: derive(#FFFFFF, 20%);
+    background-color: #FFFFFF; /* Used in the default.html file */
```

Comment:
darkTheme with a white background-color is extremely sus

Reply:
likewise for the remaining text colours. can refactor and rename the file to mainTheme or something

Reply:
UI improvement suggestions: don't use pure white and pure black for background colour and text colour respectively since it can strain your vision or something. an off-white (#111111 or #222222) and off-black colour (#EEEEEE, #DDDDDD) decreases contrast and won't hurt your eyes as much

Reply:
yessir will go and edit